src/main/java/com/epam/ta/reportportal/core/item/impl/status/ToFailedStatusChangingStrategy.java

```diff
@@ -74,9 +75,10 @@ protected void updateStatus(Project project, Launch launch, TestItem testItem, S
 
 		testItem.getItemResults().setStatus(providedStatus);
 		if (Objects.isNull(testItem.getRetryOf())) {
-			if (testItem.getItemResults().getIssue() == null && testItem.isHasStats()) {
-				addToInvestigateIssue(testItem, project.getId());
-			}
+			ofNullable(testItem.getItemResults().getIssue()).ifPresent(issue -> {
```

Comment:
this logic should not be changed:
we may have item with status "SKIPPED" and issue with types: ab, pb, etc. This issue type should not be removed if it exists

Reply:
Ok, thanks, also fixed this in ToUntestedChangingStrategy.java